src/components/side_panel/chart/gauge_chart_panel/gauge_chart_config_panel.ts

```diff
@@ -47,3 +47,9 @@ export class GaugeChartConfigPanel extends Component<Props, SpreadsheetChildEnv>
     });
   }
 }
+
+GaugeChartConfigPanel.props = {
+  figureId: String,
```

Comment:
`figureId` is only used once in all the design/config chart side panels (`LineConfigPanel`). And I'm pretty sure it's possible to replace it with props.definition in the `LineConfigPanel`.
May be worth to remove it, that's a whole 10 lines of code that we could remove 😮

Reply:
It's used a lot in Odoo, so I will check that later